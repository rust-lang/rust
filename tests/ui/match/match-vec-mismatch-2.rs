fn main() {
    match () {
        [()] => { }
        //~^ ERROR expected an array or slice, found `()`
    }
}
