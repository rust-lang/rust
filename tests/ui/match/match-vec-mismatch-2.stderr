error[E0529]: expected an array or slice, found `()`
  --> $DIR/match-vec-mismatch-2.rs:3:9
   |
LL |         [()] => { }
   |         ^^^^ pattern cannot match with input type `()`

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0529`.
